docs/pr-body/README.md

```diff
@@ -0,0 +1,43 @@
+# Pull Request Body
+
+In order for Captain's log to pick up any issues addressed in a pull request, you must include them in the pull request body.
+
+### How to get Captain's Log to pick up an issue
+
+For example:
+
+![ ](../static/pr-example.png)
+
+The linking of _#34_ in the pull request will automatically get picked up by Captain's Log. 
+
+The same goes for all of the github keywords, like "closes #34" (etc.).
+
+Linking Jira tickes is as easy as mentioning them in the ticket.
```

Comment:
You probably meant "tickets" here, not "tickes"?

Reply:
Nice catch! I gave it a once over, and still missed this!